src/app/beer_garden/requests.py

```diff
@@ -740,11 +693,11 @@ def complete_request(
     return request
 
 
-def cancel_request(request: Request) -> Request:
+def cancel_request(request_id: Request) -> Request:
     """Mark a Request as CANCELED
```

Comment:
Do we need to have a Event type for cancellation?

Reply:
Yeah, that would probably be helpful.

Reply:
Added an Event in the brewtils branch.